### DISCUSSION TITLE
MIT License

### DISCUSSION THREAD

**Discussion by kurumbus** [laravel/framework#44678] [General]
The question might sound a bit strange, but I have a weird client, and they raised a concern that laravel.com doesn't mention MIT License (or any other licensing question) anywhere. It doesn't even have a word "license"

![2022-10-21_14h59_31](https://user-images.githubusercontent.com/13556916/197180849-1a649bba-5c77-4eaa-a36d-062ae6eb6ccb.png)

It is here: https://github.com/laravel/framework/blob/9.x/README.md
But that would need explaining what is "version control", "GitHub" and a lot of other things. What are possible answers?

**Comment by Smoggert** [comment]
If the client doesn't understand the prerequisites to obtain licensing information for this kind of software. Then why is he even 'asking' about it?
Clearly the client is out of his league, and should be either informed properly by you or educate himself.

Edit: To elaborate, you will most likely be using packages as well, for which your only source of this kind of info will be github or packagist.

**Comment by kurumbus** [comment]
Well, the thing is pretty much any framework, library or CMS I could find has this info:
https://www.codeigniter.com/ (footer)
https://vuejs.org/ (footer)
https://reactjs.org/docs/how-to-contribute.html#license
https://kotlinlang.org/ (footer)
https://www.yiiframework.com/license
https://cakephp.org/ (homepage)
https://jquery.org/license/
https://wordpress.org/about/license/
https://www.joomla.org/about-joomla.html (I mean yes even Joomla)

So it's just strange. I know about the packages situation, but I'm just interested is there a reason for it or is it like, whatever?
